testdata/ingress_rg_notshared_alb/input/k8s/01-ing.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: myingress
```

Comment:
Should not this be also `shread: false` ?, at least the directory tells about "notshared".

Reply:
I think here the idea is to have one configured only with default values and another with `shared: false` and test if the ingress controller would create two LBs, one for the ingress that shares the LB and other to a RG that does not share the LB.

Maybe we could improve this test naming.